devfile-support/samples/sample-devfile.yaml

```diff
@@ -1,5 +1,6 @@
-name: "devfile example"
 schemaVersion: "2.0.0"
+metadata:
+  name: "devfile example"
```

Comment:
Should one of these samples include a `.metadata.version`?

Reply:
done in `simple-devfile.yaml`